sdk/organizations.go

```diff
@@ -1722,14 +1723,33 @@ type ResponseItemOrganizationsCreateOrganizationInventoryOnboardingCloudMonitori
 type ResponseOrganizationsReleaseFromOrganizationInventory struct {
 	Serials []string `json:"serials,omitempty"` // Serials of the devices that were released
 }
+
+type HeadLicenseID string
+
+func (h *HeadLicenseID) UnmarshalJSON(data []byte) error {
+	var str string
+	if err := json.Unmarshal(data, &str); err == nil {
+		*h = HeadLicenseID(str)
+		return nil
+	}
+
+	var num float64
+	if err := json.Unmarshal(data, &num); err == nil {
+		*h = HeadLicenseID(fmt.Sprintf("%.0f", num))
+		return nil
+	}
+
+	return fmt.Errorf("HeadLicenseID: unsupported type: %s", string(data))
```

Comment:
if string(data) == "null"
In this case it would return error ?
We should handle this

Reply:
if `headLicenseId` is `null` in the JSON response, it doesn't throw an error because `json.Unmarshal` handles that. `null` is a valid JSON value, and since `HeadLicenseID` is a string, it will be an empty string ""